www/docs/usage/config-files.md

```diff
@@ -53,28 +56,27 @@ bin = "$HOME/bin/terraform"
 version = "0.11.3"
 ```
 
-6. Run `tfswitch` and it should automatically install the required terraform version in the specified binary path
+6. Run `tfswitch` and it should automatically install the required version in the specified binary path
 
-**NOTE**
-
-1. For Linux users that do not have write permission to `/usr/local/bin/`, `tfswitch` will attempt to install terraform at `$HOME/bin`. Run `export PATH=$PATH:$HOME/bin` to append bin to PATH
-2. For Windows host, `tfswitch` need to be run under `Administrator` mode, and `$HOME/.tfswitch.toml` with `bin` must be defined (with a valid path) as minimum, below is an example for `$HOME/.tfswitch.toml` on windows
+Below is an example for `$HOME/.tfswitch.toml` on Windows:
 
 ```toml
 bin = "C:\\Users\\<%USRNAME%>\\bin\\terraform.exe"
 ```
 
-## Setting the default version using `.tfswitch.toml` file
+## Setting the default (fallback) version using `.tfswitch.toml` file
 
-The `.tfswitch.toml` file can be configured with a `default-version` attribute to configure tfswitch a particular version, if no other sources of versions are found
+If `tfswsitch` is unable to determine the version to use, it errors out.  
+The `.tfswitch.toml` file can be configured with a `default-version` attribute for `tfswitch` to use a particular version, if no other sources of versions are found
```

Comment:
This mentions the TOML, is the only way to find out about the env variable to see the seperate page about "Setting the default (fallback) version"?
I know the title emplies it's _just_ for using the .tfswitch.toml, but the fact there's an environment variable feels weird wording it in this way.

I almost feel like we should have a config guide.. and each of the configs say whether they can be set via cmd-switch, env variable or toml file. Rather than duplicates for each of the configuration methodologies

Then the title is just "## Setting the default (fallback) version" explaining the behavoir, the configuration available to overcome and then the various methods for setting the config

Reply:
Yep, I agree that we 're better off revisiting and reviewing the documentation to bring it consistent with what have changes over past year or so... Gonna be a task with its own issue/PR.